Example/Custom Integration/PaymentExampleViewController.h

```diff
@@ -0,0 +1,30 @@
+//
+//  PaymentExampleViewController.h
+//  Custom Integration
+//
+//  Created by Cameron Sabol on 10/7/19.
+//  Copyright © 2019 Stripe. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+#import <Stripe/Stripe.h>
+
+#import "BrowseExamplesViewController.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface PaymentExampleViewController : UIViewController <STPAuthenticationContext>
```

Comment:
so we can stop just copy-pasting as we add view controllers. Should definitely modernize this code more though